src/components/ProjectSearch/TextSearch.js

```diff
@@ -185,7 +185,9 @@ export default class TextSearch extends Component<Props> {
         />
       );
     } else if (status === statusType.fetching) {
-      return <div className="no-result-msg absolute-center">Loading...</div>;
+      return (
+        <div className="no-result-msg absolute-center">No results found.</div>
```

Comment:
Nicely done! One improvement would be that the `No results found` text needs to be localized through L10N. (if you search the file for `L10N` you'll see how it works)